kie-wb-common-react/boxed-expression-editor/src/hooks/Hooks.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Dispatch, RefObject, SetStateAction, useCallback, useEffect, useRef, useState } from "react";
+
+export function useContextMenuHandler(): [
+  RefObject<HTMLDivElement>,
+  string,
+  string,
+  boolean,
+  Dispatch<SetStateAction<boolean>>
+] {
+  const wrapperRef = useRef<HTMLDivElement>(null);
+
+  const [xPos, setXPos] = useState("0px");
+  const [yPos, setYPos] = useState("0px");
+  const [contextMenuIsVisible, setContextMenuVisibility] = useState(false);
```

Comment:
Little discrepancy between variable name and its function :) I can be pedantic, I know

Reply:
That's totally fine. Thank you!